CMakeLists.txt

```diff
@@ -1,36 +1,17 @@
 cmake_minimum_required(VERSION 3.4.3)
 
-project(OptSched)
 
 option(OPTSCHED_INCLUDE_TESTS "Generate build targets for the OptSched unit tests." ON)
-option(OPTSCHED_ENABLE_AMDGPU "Build the AMDGPU code. Requires that the AMDGPU target is supported." OFF)
+set(OPTSCHED_ENABLE_AMDGPU "Build the AMDGPU code. Requires that the AMDGPU target is supported." ON)
```

Comment:
This should remain as an `option()`. Changing the default value is okay, but it should be set to whatever the most common setup for our project is. Those outside the most common setup should configure cmake with `-DOPTSCHED_ENABLE_AMDGPU=OFF` (or `ON` as appropriate).

Would be good to use CMake presets, but I haven't used CMake recently enough to quickly set that up.